pages/index.tsx

```diff
@@ -38,10 +45,48 @@ export async function getStaticProps({
 export default function Home({
   products,
   categories,
-  brands
+  brands,
+  locale
 }: InferGetStaticPropsType<typeof getStaticProps>) {
+  const [orderedBrands, setOrderedBrands] = useState<Array<string>>([])
+  const [productList, setProductList] = useState<Array<Product>>(products)
```

Comment:
Pointing to the type of the original value, which is also your initial state, makes it easier to maintain. If the type of `products` would ever change, this will change as well without manually having to edit it. 

```suggestion
  const [productList, setProductList] = useState<typeof products>(products)
```

Reply:
Nice little trick thanks